### PR TITLE
Feature/environment setup

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
This PR contains

- different env files for different environments
- scripts for running android, debug builds for different environments

Uploading Screen Recording 

https://github.com/Real-Dev-Squad/mobile-app/assets/64767673/df090432-fcce-4b26-9c7f-c5a36983a462

2023-08-22 at 11.23.23 PM.mov…

https://github.com/Real-Dev-Squad/mobile-app/assets/64767673/18ff39ea-3644-461a-b08f-023fda6fa96f

### CODE REVIEW

**Review by shreya-mishra — COMMENTED** [review]
Please make this PR as WIP as prod release env has not setup

File: .gitignore [hunk 1]
```diff
@@ -36,6 +36,7 @@ node_modules/
 package-lock.json
 npm-debug.log
 yarn-error.log
+yarn.lock
```
Comment (sahsisunny) [inline .gitignore]: Why?

File: .env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ENV=production
+# please add localhost url here
+# API_URL= https://api.realdevsquad.com/
```
Comment (sahsisunny) [inline .env]: you can add the default local URL here.

File: android/app/build.gradle
Comment (sahsisunny) [inline android/app/build.gradle:1]: Why so many comments here?

**Review by sahsisunny — CHANGES_REQUESTED** [review]
This pull request is intended for staging environment, but I noticed that it contains changes that alter the context of the request. Can you explain why?

**Comment by mohammadkhizerkhan** [comment]
> This pull request is intended for staging environment, but I noticed that it contains changes that alter the context of the request. Can you explain why?

can you please elaborate here?

File: .env [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ENV=dev
+# please add localhost url here
+API_URL= http://localhost:3000/
```
Comment (sahsisunny) [inline .env]:
```suggestion
API_URL= http://localhost:3000
```

File: .env.staging [hunk 4]
```diff
@@ -0,0 +1,2 @@
+ENV=staging
+API_URL= https://staging-api.realdevsquad.com/
```
Comment (sahsisunny) [inline .env.staging]:
```suggestion
API_URL= https://staging-api.realdevsquad.com
```

File: .env.production [hunk 5]
```diff
@@ -0,0 +1,2 @@
+ENV=production
+API_URL= https://api.realdevsquad.com/
```
Comment (sahsisunny) [inline .env.production]:
```suggestion
API_URL= https://api.realdevsquad.com
```

File: src/screens/AuthScreen/AuthScreen.tsx [hunk 6]
```diff
@@ -238,7 +239,7 @@ const AuthScreen = () => {
         <Text style={AuthViewStyle.cmpnyName}>{Strings.REAL_DEV_SQUAD}</Text>
       </View>
       <View style={AuthViewStyle.btnContainer}>
-        <View style={AuthViewStyle.btnContainer}>
+        {/* <View style={AuthViewStyle.btnContainer}>
```
Comment (sahsisunny) [inline src/screens/AuthScreen/AuthScreen.tsx]: Could you please explain why you added a comment to this component?

**Comment by mohammadkhizerkhan** [comment]
> This pull request is intended for staging environment, but I noticed that it contains changes that alter the context of the request. Can you explain why?

addressed this issue

**Pull request closed without merging**